scripts/rook.sh

```diff
@@ -26,7 +26,15 @@ function deploy_rook() {
 		kubectl create -f "${TEMP_DIR}/cluster-test.yaml"
 		rm -rf "${TEMP_DIR}"
 	else
-		kubectl create -f "${ROOK_URL}/cluster-test.yaml"
+		# add "mon_warn_on_pool_no_redundancy = false" to ceph.conf if missing
+		# see https://github.com/rook/rook/pull/5925 for upstream status
+		TEMP_DIR="$(mktemp -d)"
+		curl -o "${TEMP_DIR}"/cluster-test.yaml "${ROOK_URL}/cluster-test.yaml"
+		if ! grep -q mon_warn_on_pool_no_redundancy "${TEMP_DIR}"/cluster-test.yaml; then
+			sed -i '/osd_pool_default_size =/a \    mon_warn_on_pool_no_redundancy = false' "${TEMP_DIR}"/cluster-test.yaml
```

Comment:
also `osd_pool_default_size`

Reply:
Sorry, I don't get what you mean. `osd_pool_default_size` is present in the ceph.conf for in the `cluster-test.yaml`, the sed command adds the `mon_warn_on_pool_no_redundancy` option on the line after it (with added indention).

Reply:
same as above.

Reply:
In that case, it would make sense to replace `cluster-test.yaml` with a variable. We do not do that because it will become unreadable. `osd_pool_default_size` is used only once, and there is no intention to make it configurable. It would not make the code any better.